### ISSUE TITLE
[WinUI 3 Gallery: Design-> Resources]: While navigating to ‘Segoe UI Variable' link, one side of the keyboard focus indicator is getting truncated.

### ISSUE THREAD

**Issue by sabareesanms** [microsoft/WinUI-Gallery#1928]
**Environment Details:**

**OS Version:** 24H2 (OS Build 26100.4315)

**App:** WinUI 3 Gallery

**App Version:** 2.6.0.0 (SDK 1.7)

**AT:** Keyboard

**Repro Steps:**

1. Launch the WinUI 3 Gallery app.

2. Navigate to search box using tab key.

3. Type 'Typography' in search box and hit enter.

4. Navigate to‘Segoe UI Variable' link.

5. Observe the keyboard focus boarder (outline).

**Actual Result:**

While navigating to ‘Segoe UI Variable' link, one side of the keyboard focus indicator is getting truncated.

**Expected Result:**

While navigating to ‘Segoe UI Variable' link all four sides of the keyboard focus indicator should be visible clearly to the user.

**User Impact:**

Users with low vision will get confused and face difficulty in identifying the focused element if the focus indicator is truncated or not visible clearly to the user.

**Comment by AndrewKeepCoding** [comment]
Here is how it looks:

<img width="847" height="149" alt="Image" src="https://github.com/user-attachments/assets/7352fdcf-c700-4116-9955-6db5b6023e02" />

**Comment by sabareesanms** [comment]
Check this image for reference

<img width="1311" height="692" alt="Image" src="https://github.com/user-attachments/assets/01433205-d2ba-4b96-91d4-6146cebbcde1" />
